### DISCUSSION TITLE
no .litmusconfig file after installing v0.9.0

### DISCUSSION THREAD

**Discussion by bruteMax** [litmuschaos/litmusctl#62] [Q&A] [answered]
Hi.  I installed v0.9.0 on my linux server but after trying to install a new chaos agent I get this error: 

File reading error open /root/.litmusconfig: no such file or directory

Is there a step during install that I'm overlooking?

Thanks!

**Comment by SarthakJain26** [comment, marked as answer]
@rajdas98

**Reply by bruteMax** [reply]
I created a config file via:  litmusctl config set-account --endpoint="" --username="" --password=""
